.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Do we need that in the architecture repo and if so, do we need this in this PR?

Reply:
We don't need it, I think that might have been automatically included when editing it with an IntellJ editor. I'll remove it